common/scala/src/main/scala/whisk/core/entity/Exec.scala

```diff
@@ -122,9 +122,9 @@ protected[core] case class BlackBoxExec(override val image: String, code: Option
     override val pull = image != Exec.BLACKBOX_SKELETON
 }
 
-protected[core] case class SequenceExec(code: String, components: Vector[FullyQualifiedEntityName]) extends Exec(Exec.SEQUENCE) {
-    val image = Exec.imagename(Exec.NODEJS)
-    def size = components.map(c => c.size).reduce(_ + _)
+protected[core] case class SequenceExec(components: Vector[FullyQualifiedEntityName]) extends Exec(Exec.SEQUENCE) {
+    override val image = ""  // image does not have a meaning for sequences anymore
```

Comment:
should we change the type of image then to be an option in the abstract type and make this none?
or make this throw an exception?